weatherApp/App/Data/Core Data/CoreDataStack.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  CoreData.swift
+//  weatherApp
+//
+//  Created by Dario Nikolic on 19/08/2020.
+//  Copyright © 2020 Dario Nikolic. All rights reserved.
+//
+
+import CoreData
+
+class CoreDataStack {
+    
+    public typealias CoreDataManagerCompletion = () -> ()
+    public static let weatherApp = "weatherApp"
```

Comment:
If you want to make this class "generic" and reusable outside of this app you need to remove weatherApp static property.